cli/commands/subcommands/build.js

```diff
@@ -23,7 +23,7 @@ module.exports = (program, config = {}) => {
     `
       rm -rf ${config.build.path} && 
       mkdir ${config.build.path} &&
```

Comment:
Why? At the beginning this dir could not even exist and `mkdir` will fail.

For example we want this file structure:

```
└─┬ app
  └─┬ artifacts
    └── dist
```

And there is no `artifacts` at the beginning.

So `rm -rf ./artifacts/dist` will succeed but `mkdir ./artifacts/dist` will fail with `mkdir: atrifacts: No such file or directory`.